src/argosim/plot_utils.py

```diff
@@ -291,3 +292,59 @@ def plot_uv_hist(baselines, bins=20, output_folder=None):
         plt.show()
 
     return baseline_hist
+
+
+from metrics_utils import compute_eccentricity
+
+
+def plot_beam_fit_ellipse(beam, fit_result, zoom=30):
```

Comment:
The default value for zoom is a bit arbitrary. You could set `None` as a default and in case zoom is `None` you precompute something based on the beam image, otherwise the user can still input the value they want for the zoom.